auto-pr-stage-to-master/action.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Create Release PR into master from stage'
+description: 'Create Release PR into master from stage'
+
+inputs:
+  gh-token:
+    description: 'Authorization Bearer GH Token'
+    required: true
+  body:
```

Comment:
How about naming it "create-release-pull-request" and renaming the input "body" to "description" or "details"? Don't forget about the empty line at the end;)